initialscreen.css

```diff
@@ -0,0 +1,635 @@
+    padding-top: 30px;
+}
+.picturelink {
+    width: 275px;
+    height: 175px;
+    display: inline-block;
+    background-color: #F5A7BC;
+    margin: 10px 5px 0 5px;
+}
+.picturelink.front-picture {
+    display: block;
+    position: static;
+}
+.picturelink.back-picture {
+    display: block;
+}
+.picturelink:hover .front-picture {
+    display: none;
+}
+.picturelink:hover .back-picture {
+    display: block;
+    position: static;
+}
+.front-picture {
+    background-color: #C979BA;
+    width: 100%;
+    height: 100%;
+}
+.back-picture {
+    height: 100%;
+    width: 100%;
+    background-color: #B5C7FF;
+    position: static;
+    display: none;
+}
+.title-wrapper {
+    width: 99%;
+    margin: auto;
+    text-align: center;
+}
+#my-photo {
+    background-image: url("ron-weasley-adorable-eating (1).png");
+    background-position: center;
+    width: 50%;
+    height: 150px;
+    margin: auto;
+    background-repeat: no-repeat;
+}
+.contactme {
+    width: 100%;
+    height: 650px;
+    background-image: linear-gradient( #E5EBFF, #B5C7FF, #A1B1E2 )    ;
+    margin-top: 200px;
+}
+.contact-me-title {
+    text-align: center;
+    font-size: 300%;
+}
+.contactinfo {
+    text-align: center;
+}
+
+
+
+/* @MEDIA FUNCTIONS */
+@media only screen and (max-width: 992px) {
+    body {
+        font-size: 82.6%;
+    }
+    .picturelink {
+        width: 220px;
+        height: 144px;
+    }
+    .contactme {
+        margin-top: 140px;
+        padding-top: 30px;
+    }
+    #byeblock {
+        visibility: hidden;
+    }
+}
+@media only screen and (max-width: 768px) {
+    body {
+        font-size: 64%;
+    }
+    .stickynav {
+        height: 30px;
+    }
+    .picturelink {
+        width: 176px;
+        height: 112px;
+    }
+    .content-container {
+        height: 800px;
+    }
+    .aboutme {
+        display: block;
+        float: none;
+        width: 100%;
+        padding: 0 5px 0 5px;
+        height: 50%;
+    }
+    .portfolio {
+        display: block;
+        float: none;
+        width: 100%;
+        margin-top: 50px;
+    }
+    .contactme {
+        display: block;
+        margin-top: 400px;
+    }
+    #byeblock {
+        visibility: visible;
+    }
+    #my-photo {
+        width: 50%;
+    }
+    #metitle {
+        padding-top: 20px;
+    }
+}
+@media only screen and (max-width: 576px) {
+    body {
+        font-size: 48%;
+    }
+    .stickynav {
+        height: 20px;
+    }
+    .picturelink {
+        width: 90px;
+        height: 66px;
+    }
+    .aboutme {
+        display: block;
+        float: none;
+    }
+    .portfolio {
+        display: block;
+        float: none;
+        width: 100%;
+    }
+}
+@media only screen and (max-width: 320px) {
+    body {
+        font-size: 26.67%;
+    }
+    .stickynav {
+        height: 10px;
+    }
+    .picturelink {
+        width: 73.3px;
+        height: 46.7px;
+    }
+}
+
+/*! normalize.css v8.0.0 | MIT License | github.com/necolas/normalize.css */
```

Comment:
Normalize should go in its own file, & be called before your css.

Reply:
placed it before my css